config.yaml

```diff
@@ -47,6 +47,7 @@ clusters:
         uarch: 'zen2'
         partition: 'normal'
         variables:
+          F7T_URL: "https://api.cscs.ch/hpc/firecrest/v1"
```

Comment:
this probably should not be part of this PR anymore, once changes from main have been merged

Reply:
Nope, not sure why it's showing up in the diff still (I suppose from the common ancestor...). Let me merge in main first just to make sure it's gone.

Reply:
It was actually correctly showing up in the diff. The `F7T_URL` entry was duplicated in this PR. I've removed it now. Thanks for pointing this out!